Lib/string/__init__.py

```diff
@@ -292,34 +288,33 @@ def convert_field(self, value, conversion):
             return ascii(value)
         raise ValueError("Unknown conversion specifier {0!s}".format(conversion))
 
-
-    # returns an iterable that contains tuples of the form:
-    # (literal_text, field_name, format_spec, conversion)
-    # literal_text can be zero length
-    # field_name can be None, in which case there's no
-    #  object to format and output
-    # if field_name is not None, it is looked up, formatted
-    #  with format_spec and conversion and then used
     def parse(self, format_string):
-        return _string.formatter_parser(format_string)
+        """
+        Return an iterable that contains tuples of the form
+        (literal_text, field_name, format_spec, conversion).
 
+        *literal_text* can be zero length and *field_name*
+        can be None, in which case nothing is formatted.
```

Comment:
This is misguiding. It can be read that if literal_text is zero length, then nothing is formatted. Or the if both literal_text is zero length and field_name is None, then nothing is formatted.

Actually, they were different sentences:

1. "literal_text can be zero length"
2. "field_name can be None, in which case there's no object to format and output"

Negative indentation was used in a comment to denote a continuation. The lack of punctuation created confusion.